test/test-recommend.js

```diff
@@ -0,0 +1,47 @@
+const { Recommender } = require("../lib/Recommend");
+
+const tabs = require("sdk/tabs");
+function only1Tab () {
+  let first = true;
+  for (let tab of tabs) {
+    if (first) {
+      first = false;
+      continue;
+    }
+    tab.close();
+  }
+}
+
+
+exports['test only onboard while installing'] = function (assert) {
+  assert.pass();
+};
+
+
+
+exports['test onboarding' ] = function (assert) {
+  let R;
+
+  console.log(Object.keys(assert));
+
+  // install
+  R = new Recommender();
+  console.log(R.onboarding);
+  assert.ok(!R.onboarding, "R should NOT be onboarding yet.");
+  R.start('install');
+  assert.ok(R.onboarding, "R should onboarding yet.");
+  R.destroy()
+
+  // startup
+  R = new Recommender();
```

Comment:
This second stanza should be an separate test, since it is entirely independent of the first stanza.